### PR TITLE
Indirect fontname problem

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#205 
### Changed
- `__detailed_return` parameter added to `text2art` function

#### Any other comments?
By adding this new parameter to `text2art` function, higher-level APIs can have converted font names. So warnings will be shown with the extracted font name later on.

### CODE REVIEW

File: art/art_param.py [hunk 1]
```diff
@@ -27,8 +27,10 @@
 OVERWRITE_TYPE_ERROR = "The 'overwrite' type must be bool."
 SEP_TYPE_ERROR = "The 'sep' type must be str."
 SPACE_TYPE_ERROR = "The 'space' type must be int."
+DETAILED_RETURN_TYPE_ERROR = "The '__detailed_return' type must be bool."
 ART_ENVIRONMENT_WARNING = "[Warning] '{0}' is not printable in this environment."
-FONT_ENVIRONMENT_WARNING = "[Warning] '{0}' is not printable in this environment."
```
Comment (sadrasabouri) [inline art/art_param.py:31]: I am not sure about `aprint` if we should update it, too. @sepandhaghighi what do you think?

Comment (sepandhaghighi) [reply art/art_param.py:31]: We will handle this one in another PR 💯

File: art/art.py [hunk 2]
```diff
@@ -555,6 +565,8 @@ def set_default(
     :type sep: str
     :param space: space between characters
     :type space: int
+    :param __detailed_return: flag for returning the font
```
Comment (sepandhaghighi) [inline art/art.py:568]: :param __detailed_return: flag for returning the font and the decoration

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**